tests/test_other.py

```diff
@@ -7047,7 +7047,10 @@ def test_link_response_file_does_not_force_absolute_paths(self):
     shutil.rmtree(os.path.join(current_directory, with_space))
 
     # We want only the relative path to be in the linker args, it should not be converted to an absolute path.
-    self.assertItemsEqual(link_args, [main_object_file_name])
+    if hasattr(self, 'assertCountEqual'):
```

Comment:
maybe also mention python 2 compat on this one? (again, it looks odd without an explanation)

Reply:
Oops, sorry. Also done.